tests/filecheck/transforms/riscv_cse.mlir

```diff
@@ -10,16 +10,22 @@
 riscv.assembly_section ".text" {
     %d8 = riscv.li 8 : () -> !riscv.reg<>
     %e8 = riscv.li 8 : () -> !riscv.reg<>
+
+    "test.op"(%d8, %e8) : (!riscv.reg<>, !riscv.reg<>) -> ()
 }
 
 %f8 = riscv.li 8 : () -> !riscv.reg<>
 
+"test.op"(%a8, %b8, %c8, %a7, %a7, %b7, %f8) : (!riscv.reg<>, !riscv.reg<>, !riscv.reg<>, !riscv.reg<>, !riscv.reg<>, !riscv.reg<>, !riscv.reg<>) -> ()
+
```

Comment:
Added two `test.op`s to use value and prevent CSE to rightfully CSE away the whole module